setup.py

```diff
@@ -42,17 +43,21 @@
         'tests': [
             'coveralls',
             'doubles',
-            'flake8<3',  # see https://github.com/zheller/flake8-quotes/issues/29
+            'flake8',
             'flake8-quotes',
-            'mock<1.1.0',
-            'psycopg2>=2.4.0',
+            'mock',
+            'psycopg2-binary',
             'sqlalchemy>=1.2.0',
-            'pytest>=3.0.0',
+
+            # pytest-tornado isn't compatible with pytest>=4.0.0,
+            # see https://github.com/eugeniy/pytest-tornado/pull/38
+            'pytest>=3.0.0,<4.0.0',
```

Comment:
Just for transparency:
eugeniy/pytest-tornado#38

Reply:
I pinned to <3.8 in master, because 3.9 already fails with warnings. Please merge first.

Reply:
Hello @yurishkuro,

It looks like you forgot to push your changes.

Reply:
oh, sorry, this was in jaeger-client-python. I assume the same pin would be needed here.

Reply:
> 3.9 already fails with warnings.

There are warnings, but [it doesn't fail](https://travis-ci.org/uber-common/opentracing-python-instrumentation/jobs/458209673#L763) with `pytest-3.10.1`.

Reply:
don't know what may be different here, jaeger-client-python [was failing][1] with things like
```
_________ ERROR at setup of test_trace_propagation[TCHANNEL-HTTP-True] _________
../../../virtualenv/python2.7.14/lib/python2.7/site-packages/pytest_tornado/plugin.py:171: in http_server
    http_app = request.getfuncargvalue(request.config.option.app_fixture)
E   RemovedInPytest4Warning: getfuncargvalue is deprecated, use getfixturevalue
```

[1]: https://travis-ci.org/jaegertracing/jaeger-client-python/jobs/458448824

Reply:
maybe this lib doesn't use pytest_tornado

Reply:
> don't know what may be different here, jaeger-client-python was failing with things like

I see `pytest-4.0.0` [was used](https://travis-ci.org/jaegertracing/jaeger-client-python/jobs/458448824#L780-L781) in that case.
```
============================= test session starts ==============================
platform linux2 -- Python 2.7.14, pytest-4.0.0, py-1.5.2, pluggy-0.8.0
```

Reply:
And `pytest` [was pinned](https://github.com/jaegertracing/jaeger-client-python/blob/162464ab0e3093c54ea0007a5b41b6c9f86c3b05/setup.py#L56) to `>=3.6.0` in a corresponding commit.

Reply:
I was getting the same warnings with pytest 3.9 (they are about future deprecation).

Reply:
I believe that warning is ok here. Isn't it?